core/src/main/java/org/stellar/anchor/sep31/Sep31Service.java

```diff
@@ -570,11 +571,18 @@ private static Sep31InfoResponse sep31InfoResponseFromAssetInfoList(List<AssetIn
       if (assetInfo.getSep31() != null && assetInfo.getSep31().getEnabled()) {
         boolean isQuotesSupported = assetInfo.getSep31().isQuotesSupported();
         boolean isQuotesRequired = assetInfo.getSep31().isQuotesRequired();
+        List<String> methods = assetInfo.getSep31().getReceive().getMethods();
+        AssetInfo.Field type =
+            AssetInfo.Field.builder()
+                .description("methods supported by the anchor for asset deposits")
+                .choices(methods)
+                .build();
         AssetResponse assetResponse = new AssetResponse();
         assetResponse.setQuotesSupported(isQuotesSupported);
         assetResponse.setQuotesRequired(isQuotesRequired);
         assetResponse.setMinAmount(assetInfo.getSep31().getReceive().getMinAmount());
         assetResponse.setMaxAmount(assetInfo.getSep31().getReceive().getMaxAmount());
+        assetResponse.setFields(ImmutableMap.of("type", type));
```

Comment:
In my opinion `type` is too general (i.e in SEP-12 we use `type` to refer customer type), we should use something more descriptive, like  `Payment Methods` etc.

Reply:
> In my opinion type is too general (i.e in SEP-12 type is used to refer customer type), we should use something more descriptive, maybe Payment Methods etc.

I think it's fine to call it `type` here because `SEP-6` also calls the methods, `type`. Ideally, we rename both to something like `method` but I'm not sure it's making a protocol change so we can rename a field.

> Also curious if we can get rid of the the nested fields: { types: { ... } } structure (This is also used in SEP-6 info response)? fields is mostly used to list needed KYC info, but now SEP-6 and SEP-31 both has delegate it to SEP-12

Makes sense to me. SEP-6 does this already and calls the config property, `methods`.

Reply:
> I think it's fine to call it type here because SEP-6 also calls the methods, type. Ideally, we rename both to something like method but I'm not sure it's making a protocol change so we can rename a field.

Right I also talked to Jamie about this and let's stick with `type` for now